src/platform/silabs/wifi/wf200/WifiInterfaceImpl.h

```diff
@@ -129,6 +129,13 @@ class WifiInterfaceImpl final : public WifiInterface
      *        Once the processing function is reworked, it won't be necessary anymore
      */
     bool HasNotifiedIPv4() { return mHasNotifiedIPv4; }
+
+    /**
+     * @brief Updates the IPv4 address in the Wi-Fi interface and notifies the application layer about the new IP address.
+     *
+     * @param[in] ip New IPv4 address
+     */
+    void GotIPv4Address(uint32_t ip);
```

Comment:
This API is also defined in the WiseconnectWifiInterface. Since it is defined in all implementation, it can be defined in the protected section of the main WifiInterface header as an abstract function and remove it from the implementation headers /

Reply:
Moved to the common WiFiInterface header